packages/browser-repl/src/components/utils/inspect.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const utilInspect = require('util').inspect;
+const utilInspectCustom = utilInspect.custom || 'inspect';
+
+const formatBsonType = (value): any => ({
+  [utilInspectCustom](): string {
+    return `${value._bsontype}(${(JSON.stringify(value))})`;
```

Comment:
something to consider here for formatting bson is `[EJSON.stringify](https://github.com/mongodb/js-bson#ejsonstringifyvalue-replacer-space-options)` function. If `JSON.stringify()` is satisfactory for you, then ignore!

Reply:
That should be a temporary solution, we should revisit that after we implement the supplementary types for the shell: https://docs.mongodb.com/manual/core/shell-types/